cs-couchbase/src/main/java/io/cloudslang/content/couchbase/utils/InputsUtil.java

```diff
@@ -61,11 +97,111 @@ public static String getUrl(String input) throws MalformedURLException {
         return new URL(input).toString();
     }
 
-    public static String toAppend(String prefix, String suffix, String action) {
+    public static String appendTo(String prefix, String suffix, String action) {
         if (isBlank(suffix)) {
             return prefix;
         }
 
         return prefix + "/" + suffix + UriSuffix.getUriSuffix(action);
     }
+
+    public static String getPayloadString(Map<String, String> headersOrParamsMap, String separator, String suffix, boolean deleteLastChar) {
+        if (headersOrParamsMap.isEmpty()) {
+            return EMPTY;
+        }
+
+        StringBuilder sb = new StringBuilder();
+        for (Map.Entry<String, String> entry : headersOrParamsMap.entrySet()) {
+            sb.append(entry.getKey());
+            sb.append(separator);
+            sb.append(entry.getValue());
+            sb.append(suffix);
+        }
+        if (deleteLastChar) {
+            return sb.deleteCharAt(sb.length() - 1).toString();
+        }
+
+        return sb.toString();
+    }
+
+    public static int getValidPort(String input) {
+        if (isBlank(input)) {
+            return COUCHBASE_DEFAULT_PROXY_PORT;
+        }
+
+        patternCheck(input, PORT_REGEX);
+
+        return Integer.parseInt(input);
+    }
+
+    public static String getEnabledString(String input, boolean enforcedBoolean) {
+        if (getEnforcedBooleanCondition(input, enforcedBoolean)) {
+            return valueOf(1);
+        }
+
+        return valueOf(INIT_INDEX);
+    }
+
+    /**
+     * If enforcedBoolean is "true" and string input is: null, empty, many empty chars, TrUe, tRuE... but not "false"
+     * then returns "true".
+     * If enforcedBoolean is "false" and string input is: null, empty, many empty chars, FaLsE, fAlSe... but not "true"
+     * then returns "false"
+     * This behavior is needed for inputs like: "imageNoReboot" when we want them to be set to "true" disregarding the
+     * value provided (null, empty, many empty chars, TrUe, tRuE) except the case when is "false"
+     *
+     * @param input           String to be evaluated.
+     * @param enforcedBoolean Enforcement boolean.
+     * @return A boolean according with above description.
+     */
+    public static boolean getEnforcedBooleanCondition(String input, boolean enforcedBoolean) {
```

Comment:
@victorursan  yes... I'm sure... but wasn't there when io.cloudslang.content.utils got shaped... I will digg in && replace stuff 👍